workspace/__lib__/xod-dev/ssd1306-display/ssd1306-128x64-i2c-device/patch.cpp

```diff
@@ -0,0 +1,40 @@
+#pragma XOD error_raise enable
```

Comment:
```suggestion
```

In straightforward implementations like this one, the use of `raiseError` is detected automatically. KISS.